### PR TITLE
[DOCS-6306] Identity Service clean up

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: identity-service/latest/install/k8s/index.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ The Identity Service can be deployed into a new or existing Kubernetes cluster.
 
 3. Deploy the Helm chart with a command similar to the following:
 
-    The Identity Service is deployed as part of the Alfresco infrastructure chart. 
-    Normally the infrastructure chart will be deployed as part of another product chart, 
-    such as Alfresco Content Services or Alfresco Process Services. 
+    The Identity Service is deployed as part of the Alfresco infrastructure chart. Normally the infrastructure chart will be deployed as part of another product chart, such as Alfresco Content Services or Alfresco Process Services.
+
+    As an example, the following command references the infrastructure chart on its own to deploy the Identity Service and the [ngnix-ingress](https://github.com/helm/charts/tree/master/stable/nginx-ingress).
```
Comment (anxumalo) [inline identity-service/latest/install/k8s/index.md:32]: This line is duplicated - fix in multiple versions

**Review by anxumalo — CHANGES_REQUESTED** [review]
Looks much better 👍🏼  .. some additional fixes are needed across multiple files:

- see last commit about `/tutorial/sso/kerberos.md` pages
- see inline comment about `/install/k8s/index.md` pages

File: identity-service/latest/install/k8s/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply identity-service/latest/install/k8s/index.md:32]: Fixed.

**Comment by anxumalo** [comment]
The `kerberos.md` changes and duplication in the `/k8s/index.md` pages are still missing across multiple versions

**Comment by oliveralfresco2018** [comment]
@anxumalo Those changes have been made now.

File: identity-service/latest/install/k8s/index.md (hunk 1 above)
Comment (anxumalo) [reply identity-service/latest/install/k8s/index.md:32]: It's still there AFAICS ;-)

**Comment by anxumalo** [comment]
> @anxumalo Those changes have been made now.

I can't see the updates for the `kerberos.md` files in all versions

**Review by anxumalo — CHANGES_REQUESTED** [review]

File: identity-service/latest/install/k8s/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply identity-service/latest/install/k8s/index.md]: Ok its fixed now @anxumalo the search and replace had an issue with its handler.

:-) cheers

**Review by anxumalo — APPROVED** [review]

**Pull request merged**